test/models/AuthorisedOfficialsSpec.scala

```diff
@@ -45,4 +44,29 @@ class AuthorisedOfficialsSpec extends WordSpec with MustMatchers with OptionValu
 
     }
   }
+
+  "AuthorisedOfficialsPhoneNumber object" must {
+
+    "all parameters defined" in {
+
+      val authorisedOfficialsPhoneNumber = AuthorisedOfficialsPhoneNumber(
+        daytimePhone = "01632 960 001",
+        mobilePhone = Some("01632 960 001"))
+
+      authorisedOfficialsPhoneNumber.daytimePhone mustBe "01632 960 001"
+      authorisedOfficialsPhoneNumber.mobilePhone mustBe Some("01632 960 001")
+
+    }
+
+    "middleName is not defined" in {
```

Comment:
change the middleName to alternativePhoneNumber

Reply:
resolved